Frontend/src/app/styles/display.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+    width: 100%;
+    margin: 0 auto;
+}
+
+/* Media query for desktop viewport targets screens a minimum width of 1000px. */
+@media only screen and (min-width: 1000px) {
+    .container {
+      width: calc(100%-1000px);
+      max-width: calc(2000px);
+    }
+  }
+

```

Comment:
This is just spanning my whole browser window now and I know you just wanted to add some padding. If you look how I did it in the `project-details` component it may help you out. Then you would just add in the max-width like you have here for the zoom problem.

https://github.com/dyl10s/TimeTracker/blob/master/Frontend/src/app/views/dashboard/project-details/project-details.component.scss